### DISCUSSION TITLE
A `ButtonGroup` component ?

### DISCUSSION THREAD

**Discussion by UltraInstinct05** [mantinedev/mantine#272] [Feedback]
I was looking to contribute more to `Mantine` and wanted to start with something easy, like a `ButtonGroup` component ? This seems relatively easy (& looking at other UI libraries that provide some sort of `Button`, also seem to have `ButtonGroup`).

Please let me know your thoughts on this !

**Comment by rtivital** [comment]
Hi Ashwin, it not as easy as it sounds. We need to make some internal changes in order for this to work with Tooltip, Menu and Popover. Simply stacking buttons together will not work. You can use Group component to group buttons for now to group buttons and other elements.

**Reply by UltraInstinct05** [reply]
Thanks for the explanation. Yeah, had not thought about other components whose visibility is toggled based on button click.
